qutip/solver/flimesolve.py

```diff
@@ -0,0 +1,691 @@
+    tlist = np.linspace(0, timet - dt, Nt)
+    omega = (
+        2 * np.pi
+    ) / floquet_basis.T  # Frequency dependence of Hamiltonian
+
+    total_R_tensor = defaultdict(lambda: 0)
+    for cdx, c_op in enumerate(c_ops):
+        # Transforming the lowering operator into the Floquet Basis
+        #     and taking the FFT
+        modes_table = _floquet_mode_table(floquet_basis, tlist)
+
+        c_op_Floquet_basis = np.einsum(
+            "xij,jk,xkl->xil",
+            np.transpose(modes_table.conj(), (0, 2, 1)),
+            c_op.full(),
+            modes_table,
+        )
+        c_op_Fourier_amplitudes_list = np.flip(
+            np.fft.fftshift(np.fft.fft(c_op_Floquet_basis, axis=0))
+            / len(tlist)
+        )
+
+        # Building the Rate matrix
+        delta_m = np.add.outer(-floquet_basis.e_quasi, floquet_basis.e_quasi)
+        delta_m = np.add.outer(delta_m, -delta_m)
+        delta_m /= omega
+        for l, k in product(np.arange(Nt), repeat=2):
+            delta_shift = delta_m + (l - k)
+            mask = {}
+            if time_sense <= 0.0:
+                mask[0] = delta_shift == 0
+                if not np.any(mask):
+                    continue
+                rate_products = np.multiply.outer(
+                    c_op_Fourier_amplitudes_list[l],
+                    np.conj(c_op_Fourier_amplitudes_list)[k],
+                )
+            else:
+                rate_products = np.multiply.outer(
+                    c_op_Fourier_amplitudes_list[l],
+                    np.conj(c_op_Fourier_amplitudes_list[k]),
+                )
+            included_deltas = abs(delta_shift * omega) <= abs(
+                rate_products * time_sense
+            )
+            if not np.any(included_deltas):
+                continue
+            keys = np.unique(delta_shift[included_deltas])
+            for key in keys:
+                mask[key] = np.logical_and(delta_shift == key, included_deltas)
+
+            for key in mask.keys():
+                valid_c_op_products = rate_products * mask[key]
+                I_ = np.eye(Hdim, Hdim)
+                flime_FirstTerm = np.transpose(
+                    valid_c_op_products, [0, 2, 1, 3]
+                ).reshape(Hdim**2, Hdim**2)
+                tmp = np.trace(valid_c_op_products, axis1=0, axis2=2)
+                flime_SecondTerm = np.kron(tmp.T, I_)
+                flime_ThirdTerm = np.kron(I_, tmp)
+
+                total_R_tensor[key] += flime_FirstTerm - (0.5) * (
+                    flime_SecondTerm + flime_ThirdTerm
+                )
+
+    # Turning the Rate Matrix into a super operator
+    dims = [floquet_basis.U(0).dims] * 2
+    total_R_tensor = {
+        key: Qobj(RateMat, dims=dims, copy=False)
+        for key, RateMat in total_R_tensor.items()
+    }
+
+    return total_R_tensor
+
+
+def flimesolve(
+    H,
+    rho0,
+    tlist,
+    T,
+    Nt=None,
+    c_ops=None,
+    *,
+    e_ops=None,
+    args=None,
+    time_sense=0,
```

Comment:
In the class interface, `time_sense` is a positional arguments, here it is keyword only.
But the class interface does not have `Nt`... 
The tests always jump over `Nt` to input the collapse operators.
How about having `Nt` keyword only or having it in options like it is for the class interface.